kudu/kudu.py

```diff
@@ -0,0 +1,73 @@
+import urllib2
+import json
+import argparse
+
+
+def main(servers):
+    metrics = []
+    for i in range(len(servers)):
+        server = servers[i]
+        url = "{}/metrics".format(server)
+        response = urllib2.urlopen(url)
+        data = json.loads(response.read())
```

Comment:
Check the status code of the response before reading it? Basically try to handle error scenarios.